modules/local/get_chrom_sizes.nf

```diff
@@ -12,7 +12,11 @@ process GET_CHROM_SIZES {
 //        saveAs: { filename -> saveFiles(filename:filename, publish_dir:"genome") }
 
     conda     (params.enable_conda ? "bioconda::samtools=1.10" : null)
-    container "quay.io/biocontainers/samtools:1.10--h9402c20_2"
+    if (workflow.containerEngine == 'singularity' && !params.singularity_pull_docker_container) {
+        container "https://depot.galaxyproject.org/singularity/samtools:1.13--h8c37831_0"
+    } else {
+        container "quay.io/biocontainers/samtools:1.13--h8c37831_0"
+    }
```

Comment:
Please check this syntax is used for all internal modules

Reply:
I have updated all local modules that use quay containers with the syntax above. Customized containers were made for the processes for `BAMBU` (will be updated in the next release), `DEXSeq`(also contain other packages in the process), and `GET_TEST_DATA` (couldn't find a container with `git`) are not in this syntax